dsls/sql/src/test/java/org/apache/beam/dsls/sql/planner/BeamGroupByPipelineTest.java

```diff
@@ -1,111 +0,0 @@
-import org.apache.beam.dsls.sql.schema.BeamSqlRow;
-import org.apache.beam.sdk.testing.TestPipeline;
-import org.apache.beam.sdk.values.PCollection;
-import org.junit.Test;
-
-/**
- * Test group-by methods.
- *
- */
-public class BeamGroupByPipelineTest extends BasePlanner {
-  public final TestPipeline pipeline = TestPipeline.create();
-
-  /**
-   * GROUP-BY without window operation, and grouped fields.
-   */
-  @Test
-  public void testSimpleGroupExplain() throws Exception {
-    String sql = "SELECT COUNT(*) AS `SIZE`" + "FROM ORDER_DETAILS "
-        + "WHERE SITE_ID = 0 ";
-    PCollection<BeamSqlRow> outputStream = BeamSqlCli.compilePipeline(sql, pipeline);
-  }
-
-  /**
-   * GROUP-BY without window operation, and grouped fields.
-   */
-  @Test
-  public void testSimpleGroup2Explain() throws Exception {
-    String sql = "SELECT site_id" + ", COUNT(*) " + "FROM ORDER_DETAILS "
-        + "WHERE SITE_ID = 0 " + "GROUP BY site_id";
-    PCollection<BeamSqlRow> outputStream = BeamSqlCli.compilePipeline(sql, pipeline);
-  }
-
-  /**
-   * GROUP-BY with TUMBLE window.
-   */
-  @Test
-  public void testTumbleExplain() throws Exception {
-    String sql = "SELECT order_id, site_id" + ", COUNT(*) AS `SIZE`" + "FROM ORDER_DETAILS "
-        + "WHERE SITE_ID = 0 " + "GROUP BY order_id, site_id"
-        + ", TUMBLE(order_time, INTERVAL '1' HOUR)";
-    PCollection<BeamSqlRow> outputStream = BeamSqlCli.compilePipeline(sql, pipeline);
-  }
-
-  /**
-   * GROUP-BY with TUMBLE window.
-   */
-  @Test
-  public void testTumbleWithDelayExplain() throws Exception {
-    String sql = "SELECT order_id, site_id, "
-        + "TUMBLE_START(order_time, INTERVAL '1' HOUR, TIME '00:00:01')"
-        + ", COUNT(*) AS `SIZE`" + "FROM ORDER_DETAILS " + "WHERE SITE_ID = 0 "
-        + "GROUP BY order_id, site_id" + ", TUMBLE(order_time, INTERVAL '1' HOUR, TIME '00:00:01')";
-    PCollection<BeamSqlRow> outputStream = BeamSqlCli.compilePipeline(sql, pipeline);
-  }
-
-  /**
-   * GROUP-BY with HOP window.
-   */
-  @Test
-  public void testHopExplain() throws Exception {
-    String sql = "SELECT order_id, site_id" + ", COUNT(*) AS `SIZE`" + "FROM ORDER_DETAILS "
-        + "WHERE SITE_ID = 0 " + "GROUP BY order_id, site_id"
-        + ", HOP(order_time, INTERVAL '5' MINUTE, INTERVAL '1' HOUR)";
-    PCollection<BeamSqlRow> outputStream = BeamSqlCli.compilePipeline(sql, pipeline);
-  }
-
-  /**
-   * GROUP-BY with SESSION window.
-   */
-  @Test
-  public void testSessionExplain() throws Exception {
-    String sql = "SELECT order_id, site_id" + ", COUNT(*) AS `SIZE`" + "FROM ORDER_DETAILS "
-        + "WHERE SITE_ID = 0 " + "GROUP BY order_id, site_id"
-        + ", SESSION(order_time, INTERVAL '5' MINUTE)";
-    PCollection<BeamSqlRow> outputStream = BeamSqlCli.compilePipeline(sql, pipeline);
-  }
-
-  /**
-   * Query with UDF.
-   */
-  @Test
-  public void testUdf() throws Exception {
```

Comment:
This UDF test appears to have been lost with no equivalent added. Is that on purpose?